src/explorer.ts

```diff
@@ -189,7 +193,14 @@ class KubernetesResource implements KubernetesObject, ResourceNode {
         if (this.kind === kuberesources.allKinds.pod ||
             this.kind == kuberesources.allKinds.secret ||
             this.kind == kuberesources.allKinds.configMap) {
-            treeItem.contextValue = `vsKubernetes.resource.${this.kind.abbreviation}`;
+			treeItem.contextValue = `vsKubernetes.resource.${this.kind.abbreviation}`;
+            if (this.kind === kuberesources.allKinds.pod && this.metadata.status != null) {
```

Comment:
move this logic into its own function ie. 

```
if (this.kind === kuberesources.allKinds.pod) {
     treeItem.iconPath = getIconForPodStatus(this.metadata.status);
}
```

it also appears that your editor is using tabs instead of spaces - I think we've settled on using 4 space tabs in the project.

Reply:
Will fix this, does the project use an `.editorconfig` ?